pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -404,6 +408,10 @@ func NewControllerDeployment(namespace string, repository string, imagePrefix st
 		"-v",
 		verbosity,
 	}
+	if len(imagePullSecrets) > 0 {
+		container.Args = append(container.Args, "--image-pull-secret", imagePullSecrets[0].Name)
```

Comment:
If we would go with supporting setting secrets for virt-launcher, it may make more sense to either have a dedicated config field for virt-launcher on the config and not use this flag. Just picking the first out of a list may be a little bit too surprising.